canonical-websites/www.ubuntu.com
# Update to the /core page

## Done

- Complete update to the /core page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/core](http://0.0.0.0:8001/core)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#heading=h.xnpkx4ypduk)


## Issue / Card

Fixes #4508
